### ISSUE TITLE
Formatting suggestions

### ISSUE THREAD

**Issue by rudiger-gh** [AnimatedSwine37/Find-a-Friend#10]
**Game version:** Persona 4 Golden v1.1 (REV 2033)
**Mod version:** Find a Friend (v1.5)

These aren't bugs, but just suggestions for your consideration. I'd love to hear your thoughts on these.

![P4G 2020-12-27 12-48-57](https://user-images.githubusercontent.com/53123052/103173092-5ffe0a80-4850-11eb-88b1-dbbc777fcd09.png)

### Change "Lvl" to "Rank"
In the base-game, progression of social links is only ever referred to with "ranks". Mentions of "levels" are used exclusively in the context of Personas and battling as far as I can tell. Changing the phrasing throughout the mod should allow your menus to appear better integrated with the rest of the game.

### Using pages for long lists of available social links
When a large number of social links are available to hang out with, the "Find a Friend" a menu can start looking cluttered. Normally, no menu in the game has more than about six options before it starts using pages. You will never see a menu of this type reach the full height of the display.

In "Find a Friend", I've seen as many as nine options in one long list. Please can you consider the use of pages? Perhaps you could approach this as Tekka and I did for "Dungeon Fast Travel" and others:
- Create pages of descending length with the maximum number of options per-page being no more than about 8 (9 touches the top of the screen).
- Assuming the maximum number of options that could possibly be presented to the player at one time is 9, I think you should start paging after 5. This gives you a good, clean amount of space between the menu and top of the screen and will probably look neatest.

e.g. [Page 1]
> Ai (Rank 3+)
> Ayane (Rank 5+)
> Chie (Rank 6+)
> Kanji (Rank 5+)
> Kou (Rank 6+)
> Someone else
> Nobody

[Page 2]
> Naoki (Rank 1+)
> Yosuke (Rank 6+)
> Yukiko (Rank 6+)
> Someone else
> Nobody

The reason I've used "Someone else" and "Nobody" in this example, is to follow conventions elsewhere in the game. Here you can see these options being used in a similar menu for the shrine:

![P4G 2020-12-27 17-02-58](https://user-images.githubusercontent.com/53123052/103176291-639d8b80-4868-11eb-99d4-77477185dc92.png)

## Change "X is already nearby" messages to, "X is nearby."
Super minor, but "already" is redundant in this context.

![P4G 2020-12-27 17-30-34](https://user-images.githubusercontent.com/53123052/103176445-7fedf800-4869-11eb-9ae3-a245f977d982.png)

Thanks!

**Comment by AnimatedSwine37** [comment]
Thanks for the suggestions.
1. I agree that 'rank' fits better than 'level', I suppose I chose level as in the flow scripts the function to check their level is CHECK_SL_LEVEL and I just went with it.
2. The mod actually already has a pages feature which has been there since the initial release. It seems that I broke it with 1.5 as I must have missed changing some numbers when I added Dojima and Nanako. I should have that fixed soon. (That's what the 'others' menu you're seeing in #9 is meant to be)
3. I'd never actually used the shrine and seen 'nobody' and 'someone else' as options but I do agree they fit better.
